tests/unit/test_likelihoods.py

```diff
@@ -123,7 +118,8 @@ def test_gaussian_log_likelihood_known_sigma(self, problem_name, request):
         grad_result, grad_likelihood = likelihood.evaluateS1(np.array([0.5]))
         assert isinstance(result, float)
         np.testing.assert_allclose(result, grad_result, atol=1e-5)
-        assert np.all(grad_likelihood <= 0)
+        # Since 0.5 < ground_truth, the likelihood should be increasing
+        assert grad_likelihood >= 0
```

Comment:
This is interesting, do you have an idea on why this assertion flipped?

Reply:
The previous problem was inaccurate, I have put the reason for the sign in the comment on the line above.

Reply:
By problem you mean the actual `pybop.problem`, right?

Reply:
I believe the initial state of the simulation did not match the initial state of the problem, so the optimal values were different from the ground truth.